Control coverage/Feature/AVDHostPool.md

```diff
@@ -156,3 +158,113 @@ properties.securityRules<br />
 
 ___ 
 
+## Azure_AVD_Audit_Enable_HostPool_Diagnostic_Settings
+ 
+
+### Display Name 
+Diagnostics logs must be enabled for AVD Host pool VMs.
```

Comment:
if portal has diagnostic in it, why we kept diagnostics at some place?

Reply:
This is same as the display name in control JSON. In that case we might need to update the display name in that case in control JSON.

Reply:
Will update here.